cmd/admin-heal.go

```diff
@@ -96,20 +96,36 @@ var adminHealCmd = cli.Command{
 
 // healMessage container to hold repair information.
 type healMessage struct {
-	Status string             `json:"status"`
-	Bucket string             `json:"bucket"`
-	Object *madmin.ObjectInfo `json:"object"`
-	Upload *madmin.UploadInfo `json:"upload"`
+	Status string                  `json:"status"`
+	Result madmin.HealObjectResult `json:"result"`
```

Comment:
@vadmeste I have a [PR](https://github.com/minio/minio/pull/4035) that renames HealObjectResult to HealResult in pkg/madmin.